.travis.yml

```diff
@@ -1,8 +1,22 @@
-language: bash
-script: true
+---
+language: python
+python: 2.7
+cache: pip
 sudo: required
```

Comment:
Unless we start using docker in Travis CI, we don't actually need sudo. If we remove this line, builds will speed up.